Sources_v3/StreamChatUI/ChatChannel/ChatReplyBubbleView.swift

```diff
@@ -0,0 +1,159 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatReplyBubbleView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
+    // MARK: - Properties
+    
+    public var avatarViewWidth: CGFloat = .zero
+    public var attachmentPreviewWidth: CGFloat = .zero
```

Comment:
I find this pattern very confusing. Here it says `.zero` but in `defaultAppearance` it' set to 24 🤔

Reply:
Yeah, can agree.
Should we put 24 while declaring the variable or make it optional?